uw_sws/models.py

```diff
@@ -1448,6 +1448,13 @@ class Course(models.Model):
     course_title_long = models.CharField(max_length=50)
     course_campus = models.CharField(max_length=7)
     course_description = models.TextField()
+    gen_ed_req_diversity = models.NullBooleanField()
```

Comment:
we may want to avoid using NullBooleanField, use BooleanField with default None.